Python/thread.c

```diff
@@ -114,12 +114,9 @@ PyThread_set_stacksize(size_t size)
 #endif
 }
 
-#ifndef Py_HAVE_NATIVE_TLS
-/* If the platform has not supplied a platform specific
-   TLS implementation, provide our own.
 
-   This code stolen from "thread_sgi.h", where it was the only
-   implementation of an existing Python TLS API.
+/* If the platform has not supplied a platform-specific thread-local
+   storage implementation, CPython does not provide thread feature.
```

Comment:
I don't understand this comment. If you require thread support, CPython compilation fails with the above #error. It doesn't disable thread support for you. I suggest to remove this comment.